### PR TITLE
Add views and helpers for global activities individual cards [SCI-2994][SCI-3024]

### PR DESCRIPTION
**Pull request by okriuchykhin**
Jira ticket: [SCI-2994](https://biosistemika.atlassian.net/browse/SCI-2994)

### What was done
Added views and helpers for individual activity cards

### CODE REVIEW

File: app/assets/javascripts/global_activities/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      $('#' + $(this)
+        .attr('id') + '-button').find('.fas').removeClass('fa-caret-down');
+      $('#' + $(this)
+        .attr('id') + '-button').find('.fas').addClass('fa-caret-right');
+    });
+    $('.activities-group').on('show.bs.collapse', function() {
+      $('#' + $(this)
+        .attr('id') + '-button').find('.fas').removeClass('fa-caret-right');
+      $('#' + $(this)
+        .attr('id') + '-button').find('.fas').addClass('fa-caret-down');
```
Comment (aignatov-bio) [inline app/assets/javascripts/global_activities/index.js]: Maybe simplify it by adding data attribute to element and use add/remove class action in one line:

$(this.dataset.buttonLink).find('.fas').removeClass('fa-caret-down').addClass('fa-caret-right')

**Comment by urbanrotnik** [comment]
For now, everything looks good to me.

File: app/assets/javascripts/global_activities/index.js (hunk 1 above)
Comment (okriuchykhin) [reply app/assets/javascripts/global_activities/index.js]: Changed

**Comment by aignatov-bio** [comment]
I merged PR with new layout for global activities. Can you update css and resolve conflicts ?

**Pull request merged**